app/javascript/packs/postcodesHelper.js

```diff
@@ -3,52 +3,58 @@
 $(document).ready(() => {
   const clearPostcodeError = () => {
     $("#error-postcode").html("");
+    $("#error-postcode").hide();
+  };
+
+  const showPostcodeError = (error) => {
```

Comment:
DRY this out slightly, now we're using an alert we need to hide the element when there is no error